portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -50,7 +50,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int numIteratedComments = 0;
     for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
-      String comment = (String) entity.getProperty("comment");
+      String username;
+      if (entity.getProperty("username") == null) {
+          username = "anonymous";
+      }
+      else {
+          username = (String) entity.getProperty("username");
+      }
```

Comment:
Since this is duplicated below, probably worth refactoring out to a helper method, e.g. `getUsername(entity)`, and/or something more generic like `getPropertyOrDefault(entity, PROPERTY_USERNAME, "anonymous")`